.github/workflows/release-package.yml

```
@@ -0,0 +1,39 @@
+name: Release NPM package
+
+on:
+    pull_request:
```

if you add this, you can also trigger the release manually eventually, if you would be pointing, for example, to an integration branch instead of develop. :)

```suggestion
    workflow_dispatch:
    pull_request:
```

---

Thx, This would be a good improvement for the future.
However, for the time being, I don’t think the tool would be compatible.